### ISSUE TITLE
The Web Content Accessibility Guidelines (WCAG): Missing space between two sentences.

### ISSUE THREAD

**Issue by alejandro83457** [TheOdinProject/curriculum#27355]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

First paragraph in "Web Content Accessibility Guidelines" is missing a space between the two sentences.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-the-web-content-accessibility-guidelines-wcag#web-content-accessibility-guidelines

### (Optional) Discord Name

alxandrius

**Comment by Churtified** [comment]
I can make the quick fix :)
